app/editor/src/features/content/hooks/useTab.tsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import { useChannel } from 'store/hooks';
 
+let _window: Window | null = null;
```

Comment:
Now the tab is a singleton which allows us to keep track when we navigate to another page.